autoendpoint/src/routers/apns/router.rs

```diff
@@ -251,15 +302,13 @@ impl Router for ApnsRouter {
             .get("rel_channel")
             .and_then(Value::as_str)
             .ok_or(ApnsError::NoReleaseChannel)?;
+        // XXX: We don't really use anything that is a numeric here, aside from
```

Comment:
Not really an XXX: (broken/fixmeish) comment IMO

```suggestion
        // NOTE: We don't really use anything that is a numeric here, aside from
```

Reply:
Actually... It is a FIXME, specifically the third line:

```
// Once we're off of DynamoDB, we might want to kill the map.
```
That's a note to future us that we probably want to recheck if we need to do this, right?

Reply:
Ah, could do FIXME but XXX works